addons/rose/addon/components/rose/form/checkbox/index.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="rose-form-checkbox{{if @error " error"}}">
+<div class="rose-form-checkbox{{if @error " error"}} {{if @inline "inline"}}">
```

Comment:
I'd follow the pattern with the error class and include the space in the conditional string just to keep the markup tidy.

Reply:
sure, I will fix this and the formatting